src/core/gosling.schema.ts

```diff
@@ -766,18 +764,18 @@ export interface ChannelValue {
 export type AxisPosition = 'none' | 'top' | 'bottom' | 'left' | 'right';
 export type FieldType = 'genomic' | 'nominal' | 'quantitative';
 export type ValueExtent = string[] | number[];
-export type GenomicDomain = DomainInterval | DomainChrInterval | DomainChr | DomainGene;
```

Comment:
This was never supported, so let's remove this and add back whenever we support it.

Reply:
I agree with this decision. Genes are ultimately entities which correspond to intervals of genomic coordinates. There are many other genomic features which have this same property, and I don't see a motivating case to add specific support for "genes".

Reply:
It should be simple enough to map _any_ table of features which are defined by to genomic coordinates (other columns in the table) with the current API.